presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/ColumnMapping.java

```diff
@@ -24,6 +24,8 @@
 
 public final class ColumnMapping
 {
+    public static final UnaryOperator<Domain> DISABLE_PUSHDOWN = domain -> Domain.all(domain.getType());
```

Comment:
`DISABLE_PUSHDOWN` -> `PUSHDOWN_DISABLED`?

Reply:
i think it would be appropriate for an enum option.